scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -587,6 +589,42 @@ class ScioContext private[scio] (val options: PipelineOptions,
       }
     }
 
+  /**
+    * Get an SCollection for a CloudSql query
+    * @group input
+    */
+  def cloudSqlSelect[T: ClassTag](readOptions: JdbcReadOptions[T]): SCollection[T] = {
```

Comment:
Hmm, I thought to have specific API to read and write from cloudSql, that make it align with other APIs we have for google databases like bigquery, bigtable , datastore etc. But this doesn't do any different than jdbcSelect. If it doesn't make sense to have a separate wrapper API for cloud sql we can remove it.